packages/interweave/src/Parser.js

```diff
@@ -441,6 +441,21 @@ export default class Parser {
           return;
         }
 
+        // Apply transformation if available
+        const children = this.parseNode(nextNode, config);
+        const transformed = transform ? transform(nextNode, children, config) : null;
+        if (transformed === null) {
```

Comment:
Is this a way to programmatically remove nodes?

Reply:
Yep, following the same pattern as react-html-parser's return format: `undefined` = use default handling, `null` = skip node, anything else = insert to tree

Reply:
As it stands, this would abort early without rendering the children IF `transform` is not defined.

Reply:
Oh good point, that should be `transform ? ... : undefined` instead